src/make.ts

```diff
@@ -181,23 +181,19 @@ export function makeString(
   };
 }
 
-function checkNumber(value: any): Make<number> {
-  if (typeof value !== 'number') {
-    return getErrorWithValueMsg('expected a number', value);
-  }
-  return Make.ok(value);
-}
-
-export function makeNumber(value?: number): Maker<number, number> {
```

Comment:
I checked and oats never generates `makeNumber` by giving it a parameter, so we get simplify this code

Reply:
note even for enums? good catch